sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodShouldBeNameAccordingToSynchronicity.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;

```

Comment:
Please add a test for local functions. Fine if they are not covered by the rule (don't think they are now), but good to have it documented.

Reply:
They are not and they can't be (this will require C# 7 for the analyzer).